lib/src/analyzers/unused_files_analyzer/unused_files_analyzer.dart

```diff
@@ -95,6 +96,14 @@ class UnusedFilesAnalyzer {
     }).toSet();
   }
 
+  Future<void> deleteAllUnusedFiles(
```

Comment:
@incendial why this is async function? I see only sync methods.

Reply:
Fixed